doc/source/guide/data_types/maps.rst

```diff
@@ -2,72 +2,70 @@
+instruction on how to download the sample data.
+
+The variable my_map is a :ref:`map` object. To create one from a
+local FITS file try the following::
 
     my_map = sunpy.map.Map('/mydirectory/mymap.fits')
 
-SunPy automatically detects the type of file (e.g. FITS), what instrument it is
-associated with (e.g. AIA, EIT, LASCO) and will automatically look in the appropriate places for the FITS
-keywords it needs to interpret the coordinate system. If the type of FITS file
-is not recognized then SunPy will try some default FITS keywords and return a GenericMap but results
+SunPy should automatically detects the type of file (e.g. FITS), what instrument it is
+associated with (e.g. AIA, EIT, LASCO) and will automatically look in the
+appropriate places for the FITS keywords it needs to interpret the coordinate
+system. If the type of FITS file is not recognized then SunPy will try some
+default FITS keywords and return a `~sunpy.map.GenericMap` but results
 may vary. SunPy can also create maps from the jpg2000 files from
-`helioviewer.org <http://helioviewer.org/>`.
+`helioviewer.org <http://helioviewer.org/>`_.
 
 2. Creating Custom Maps
 -----------------------
-It is also possible to create maps using custom data from a simulation for example. To do this you
-need to provide `Map()` with both the data array as well as some basic meta information. If no
-header is given then some default values as assumed. Here is a simple example::
+It is also possible to create maps using custom data (e.g. from a simulation).
+To do this you need to provide `~sunpy.map.map_factory.MapFactory` with both the data array as
+well as some basic meta information. If no header is given then some default
+values as assumed. Here is a simple example::
 
     import numpy as np
     data = np.arange(0,100).reshape(10,10)
     header = {'cdelt1': 10, 'cdelt2': 10, 'telescop':'sunpy'}
     my_map = sunpy.map.Map(data, header)
 
-The format of the header follows the FITS standard.
+The keys in the header follows the `FITS standard <http://fits.gsfc.nasa.gov/fits_dictionary.html>`_.
 
 3. Inspecting maps
 ------------------
-A map contains a number of data-associated attributes. To get a quick look at your map simply
-type::
+A map contains a number of data-associated attributes. To get a quick look at
+your map simply type::
 
     import sunpy.data.sample
     my_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
     my_map
 
 This will show a representation of the data as well as some of its associated
-attributes. A number of other attributes are also available, for example the date,
-exposure time, map center, xrange, yrange
-other::
+attributes. A number of other attributes are also available, for example the
+`~sunpy.map.GenericMap.date`, `~sunpy.map.GenericMap.exposure_time`,
+`~sunpy.map.GenericMap.center`, `~sunpy.map.GenericMap.xrange`,
+`~sunpy.map.GenericMap.yrange` and others::
```

Comment:
you could link to the attributes section of the `GenericMap` documentation here.